ci/cron/daily-compat.yml

```diff
@@ -52,6 +46,31 @@ jobs:
                $(Slack.team-daml)
         displayName: report
         condition: always()
+
+  - job: compatibility
+    timeoutInMinutes: 240
+    pool:
+      vmImage: macos-10.14
```

Comment:
The `matrix` `strategy` only allows for substituting values. Since here we need to change the key (`name` vs. `vmImage`), we need to duplicate the entire job.

Reply:
You might be able to use an `if` on the variable you set in the matrix but given that this is hopefully temporary, I’m perfectly happy just duplicating it.